src/raspbian/10/helix/arm32v6/Dockerfile

```diff
@@ -1,3 +1,6 @@
+# dummy FROM to establish a dependency
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-10-crossdeps AS dependency
```

Comment:
Ok, that works. Our infrastructure does support being able to define dependencies like this in the manifest. But your solution is good enough since we don't care about the Dockerfile looking nice and pretty like we do for our official .NET Dockerfiles.

Reply:
The tooling is parsing the comments, and trying to download a base image named `to` because of `FROM to establish a dependency` 🙃